src/main/scala/uk/gov/hmrc/play/it/MicroServiceEmbeddedServer.scala

```diff
@@ -70,65 +66,6 @@ trait ExternalServiceOrchestrator extends StartAndStopServer {
   }
 }
 
-trait EmbeddedServiceOrchestrator extends ResourceProvider with StartAndStopServer {
```

Comment:
I assume none @frankoid probably removed it as there are no references to it in all of the MDTP codebase.
Is there a specific requirement to retain this..?